### PR TITLE
Updates for arm64 builds.

### PR DESCRIPTION
**Pull request by glevand**

### CODE REVIEW

File: profiles/coreos/arm64/package.accept_keywords [hunk 1]
```diff
@@ -53,6 +53,8 @@ net-fs/nfs-utils ~arm64
 #=sys-apps/smartmontools-9999 **
 =sys-apps/smartmontools-6.4 **
 
+=app-misc/jq-1.4-r1 **
+=net-misc/curl-7.45.0 ~arm64
```
Comment (mischief) [inline profiles/coreos/arm64/package.accept_keywords]: are these not sufficient?

https://github.com/coreos/coreos-overlay/blob/master/profiles/coreos/base/package.accept_keywords#L59
https://github.com/coreos/coreos-overlay/blob/master/profiles/coreos/base/package.accept_keywords#L95

**Comment by glevand** [comment]
@mischief  I had been using a partially updated source base.  The upstream profile works OK for curl and jq so I removed those. I did need to add unstable bison for arm64.  I also updated the arm64 kernel config to enable CONFIG_IFB=y.

**Comment by mischief** [comment]
lgtm

**Comment by glevand** [comment]
Rebased to coreos-kernel-4.4.4.  Will commit after final test.

**Comment by mischief** [comment]
thanks!

**Pull request merged**